### PR TITLE
Add index for performance

### LINKED ISSUE #648: Optimize table
|Query|Full Table Scan|Executed (#)|Errors (#)|Warnings (#)|Total Time|Max Time|Avg Time|
|:---------:|:------:|:------:|:------:|:------:|:------:|:------:|:------:|
|SELECT * FROM `oc_termsofservice_terms` WHERE `country_code` = ? |*|14815675|0|0|87661123471.74|8912379.29|5916.78
|SELECT * FROM `oc_termsofservice_sigs` WHERE `user_id` = ? |*|14778119|0|0|315075015116.81|6408551.81|21320.37

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #648

### CODE REVIEW

File: lib/Migration/Version1000Date20211209135602.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
```
Comment (ChristophWurst) [inline lib/Migration/Version1000Date20211209135602.php:3]: license header?

**Review by ChristophWurst — APPROVED** [review]

File: lib/Migration/Version1000Date20211209135602.php (hunk 1 above)
Comment (skjnldsv) [reply lib/Migration/Version1000Date20211209135602.php:2]: Added! :)

File: lib/Migration/Version1000Date20211209135602.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		if ($schema->hasTable('termsofservice_sigs')) {
+			$table = $schema->getTable('termsofservice_sigs');
+			$table->addIndex(['user_id'], 'tos_sigs_user_id');
```
Comment (nickvergessen) [inline lib/Migration/Version1000Date20211209135602.php:46]: https://github.com/nextcloud/terms_of_service/blob/master/lib/Db/Mapper/SignatoryMapper.php#L106

Add one for the terms as well?

**Pull request merged**